src/interface_adapter/rooms/LoadRoomsController.java

```diff
@@ -0,0 +1,19 @@
+package interface_adapter.rooms;
+
+import entities.auth.User;
+import use_case.rooms.LoadRoomsInputBoundary;
+import use_case.rooms.LoadRoomsInputData;
+
+public class LoadRoomsController {
+  final LoadRoomsInputBoundary loadRooms;
+
+  public LoadRoomsController(LoadRoomsInputBoundary roomsUseCaseInteractor) {
+    this.loadRooms = roomsUseCaseInteractor;
+  }
+
+  public void loadRooms(User user) {
+    LoadRoomsInputData roomsInputData = new LoadRoomsInputData(user);
+    // Example request to send message
```

Comment:
delete I think

Reply:
Fixed the comment in the latest commit, thanks!